tests/test_deep_copy.c

```diff
@@ -2,7 +2,12 @@
 #include <stdlib.h>
 #include <stddef.h>
 #include <string.h>
+#ifdef NDEBUG
+#undef NDEBUG
 #include <assert.h>
+#else
```

Comment:
Why not put the assert.h after the undef NDEBUG, so you don't need duplication nor a #else ?

Reply:
My fault, commit it too fast without thinking, 0.0